antenna/throttler.py

```diff
@@ -287,6 +301,13 @@ def match_unsupported_product(throttler, data):
 
 #: Ruleset for Mozilla's crash collector throttler
 MOZILLA_RULES = [
+    # If it's got an old build id, reject it now
+    Rule(
+        rule_name="has_old_buildid",
+        key="*",
+        condition=match_old_buildid,
+        result=REJECT,
+    ),
```

Comment:
Placing this first is interesting in that this might match all the `has_hangid_and_browser` crash reports and the `infobar_is_true` crash reports. If that's true, we will be able to remove those rules.